packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -49,47 +49,23 @@ export default {
   methods: {
     close() {
       this.$emit("close");
```

Comment:
wdyt about supporting `visible.sync`?

```suggestion
      this.$emit("update:visible", false);
```

or we can handle `v-model` with visible prop...

Reply:
Can you clarify what do you mean by `visible.sync`?

`v-model` sounds nice, but if I want to be independent of any trigger elements would it work?
I mean, if I would like to use `v-model` on modal, it should be a child of a `button` that will open this modal, right?

Reply:
I meant supporting [sync modifier](https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier) `<sf-model visible.sync="visible"/>`, but it's not necessary if using `v-model` :smile:

Reply:
Hi @leomp12 I've took over @michasik task and made a pull request with all your requested changes - https://github.com/DivanteLtd/storefront-ui/pull/459 Would be great If you could have a look at it :)